linters/djlint/plugin.yaml

```diff
@@ -0,0 +1,36 @@
+version: 0.1
+lint:
+  definitions:
+    - name: djlint
+      files: [html]
+      runtime: python
+      package: djlint
+      commands:
+        - name: lint
+          output: regex
+          run: djlint ${target} --quiet
+          success_codes: [0, 1]
+          batch: true
+          cache_results: true
+          parse_regex:
+            ((?P<path>.*):(?P<line>-?\d+):(?P<col>-?\d+) - (?P<code>\w\d+) - (?P<message>.*))
+        - name: fmt
+          output: rewrite
+          run: djlint ${target} --quiet --reformat
+          success_codes: [0, 1]
+          batch: true
+          cache_results: true
+          formatter: true
+          in_place: true
+      direct_configs: [.djlintrc]
+
+      affects_cache:
+        - .flake8
+        - pyproject.toml
+        - setup.cfg
+        - tox.ini
+      issue_url_format: https://www.djlint.com/docs/linter/#rules
+      known_good_version: 1.19.16
```

Comment:
Please add `is_recommended: false` to the linter's definition. This is the root of the repo_test failure because it's causing djlint to be auto-enabled as is with its config. You can make sure the repo_tests are passing locally by running `npm test tests/`

Reply:
Shouldn't it be recommended though?
I updated the PR which hopefully won't break things, but let me know if that's not ideal for how djlint works.

Reply:
Sorry I should've explained this a bit more. Right now the logic for whether trunk auto-enables a linter during `trunk init` (or auto-suggest it during `trunk upgrade`) essentially boils down to the following logic:
1. If the linter has a direct config in its definition folder (like you have here with .djlintrc) and `is_recommended: true (default)`, auto-enable.
2. If the linter has `good_without_config: true` and `is_recommended: true (default)`, auto-enable.
3. Otherwise, don't auto-enable.

We're planning to improve the naming and interaction of these fields, but as is, djlint would be auto-enabled for users. Usually we give linter integrations at least a couple weeks to marinate/stabilize before we auto-suggest them for all of our users. Especially if djlint and prettier might step on each other, that's another reason we should probably hold off.

So can you revert your `config_check_test.ts` change and add `is_recommended: false`? After dogfooding for a bit we can revisit auto-enabling it. Sorry for the confusion!